leases/management/commands/assign_ws_lease_stickers.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management import BaseCommand
+
+from applications.enums import ApplicationAreaType
+from leases.enums import LeaseStatus
+from leases.models import WinterStorageLease
+from leases.stickers import get_next_sticker_number
+
+
+class Command(BaseCommand):
+
+    help = "Assign stickers for existing paid WS leases"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Assigning stickers for existing paid WS unmarked leases")
+
+        paid_ws_leases = WinterStorageLease.objects.filter(
+            status=LeaseStatus.PAID, sticker_number=None
+        )
```

Comment:
You can add the filter for unmarked applications here
```suggestion
        paid_ws_leases = WinterStorageLease.objects.filter(
            application__area_type=ApplicationAreaType.UNMARKED,
            status=LeaseStatus.PAID,
            sticker_number=None
        )
```